### PR TITLE
[VAN-126987] null schema fix tests

### PR DESCRIPTION
**Pull request by Shmalikov**
Added several integration tests to cover issued methods.

### CODE REVIEW

File: tests/integration/visier_api_data_in/test_data_intake_api.py
Comment (hareskoug) [inline tests/integration/visier_api_data_in/test_data_intake_api.py:1]: How are these changes addressing null schemas?

Comment (Shmalikov) [reply tests/integration/visier_api_data_in/test_data_intake_api.py:1]: This PR includes a file, `test_direct_data_intake_api.py`, that isn't directly related to the task. I created it while exploring the differences between `DataIntake` and `DirectDataIntake`, and decided to keep the test.

File: tests/integration/visier_api_administration/test_production_versions_api.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        self.assertGreater(len(versions_response_dto.published_versions), 1)
+
+        operation_request_dto = ProductionVersionAPIOperationRequestDTO(
+            operation='rollBackTo'
+        )
+        operation_response_dto = self.api.post_production_version(versions_response_dto.published_versions[1].id,
+                                                                  operation_request_dto)
+        self.assertIsNotNone(operation_response_dto)
+        self.assertIsNotNone(operation_response_dto.roll_back_to)
+
```
Comment (hareskoug) [inline tests/integration/visier_api_administration/test_production_versions_api.py:37]: I know we're testing the SDK and DTO handling, but I'm wondering if it actually makes sense to call get production version again after the rollback to ensure that the version was actually updated.

**Review by hareskoug — APPROVED** [review]

File: tests/integration/visier_api_administration/test_production_versions_api.py (hunk 1 above)
Comment (Shmalikov) [reply tests/integration/visier_api_administration/test_production_versions_api.py:37]: Rollback id is not same, so there is no obvious way to make it.

**Pull request merged**